### PR TITLE
Add Basic UI to OOP School Library Console App

### PR DESCRIPTION
**Pull request by MarcoDDM**
**Description:**

This pull request adds a basic user interface (UI) to the existing Object-Oriented Programming (OOP) school library console app. The UI allows users to interact with the application more conveniently and follows the requirements specified in the project description.

**Changes Made:**
- Created `app.rb` file to serve as the console app entry-point.
- Implemented methods in `app.rb` to list all books, list all people, create a person (teacher or student), create a book, create a rental, and list all rentals for a given person ID.
- Defined the `main` method in `main.rb`, which acts as the entry point for the console app.
- Provided users with a list of options to perform various actions.
- Allowed users to choose an option and provided parameter input when necessary.
- Added a way to quit the app gracefully.
- Added screenshot for the code reviewer convenience
ui

Thank you for taking the time to review my code! Your feedback will be valuable in improving the application.

### CODE REVIEW

File: main.rb
Comment (IngDavidTM) [inline main.rb:1]: - [ ] You've done an excellent job fulfilling the project requirements :+1:. Nevertheless, during my attempt to create a new individual, book, or rental, I encountered a few errors, which are demonstrated in the video provided below. I kindly request that you look into these issues so the next reviewer can assess its functionality accurately :+1:.


https://github.com/MarcoDDM/school_library/assets/105117832/d3c76fca-1dc6-4baa-b7d4-91533b6936b4

**Review by IngDavidTM — CHANGES_REQUESTED** [review]
## Hi @MarcoDDM 🙋‍♂️, 

Good job so far!
You have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!



## Top highlight 🎯
- The Pull Request follows the GitFlow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 

## Suggested changes

- [x] _Check the comments under the review._ ⬇

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.rb [hunk 1]
```diff
@@ -56,6 +77,20 @@ def create_rental(person_id, book_title)
...
+    book_title = gets.chomp
+    create_rental(person_id, book_title)
+  end
+
+  def list_rentals_for_person_from_input
+    puts 'Enter person ID:'
+    person_id = gets.chomp.to_i
+    list_rentals_for_person(person_id)
+  end
+
```
Comment (Abenezer-Tilahun) [inline app.rb:93]: - [ Optional ] When adding a rental, the system currently requests the user to insert their personal ID. However, upon reviewing the list of all people, it appears that the personal IDs are not visible, causing confusion for the user. To address this issue and minimize confusion, I recommend making the personal IDs visible in the list of all people interface. This way, users can easily find and enter their personal ID when adding a rental, ensuring a successful rental 
creation process.
rental

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @MarcoDDM  
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Git-Flow.
✔️ Professional README  
✔️ Good Commit Messages
✔️ Good job for Implemented Suggested changes.

## Optional suggestions 
 
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

**Pull request merged**